.circleci/config.yml

```diff
@@ -3,13 +3,36 @@ version: 2.1
 commands:
   install-python-dev-tools:
     steps:
+      # See: https://circleci.com/docs/ja/2.0/language-python/#%E4%BE%9D%E5%AD%98%E9%96%A2%E4%BF%82%E3%81%AE%E3%82%AD%E3%83%A3%E3%83%83%E3%82%B7%E3%83%A5
+      - restore_cache:
+          key: &cache-key-pipenv cache-pipenv-{{ .Environment.CIRCLE_JOB }}-{{ .Environment.CIRCLECI_CACHE_VERSION }}-{{ checksum "Pipfile" }}
```

Comment:
`Pipfile.lock` should be added to the repository, and use the checksum of it.

Reply:
https://github.com/Idein/actfw-core/pull/2/commits/76d8f8c4c887d75296a431455777cc1240c033f4